manifests/o/Oven-sh/Bun/1.1.0/Oven-sh.Bun.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Oven-sh.Bun
+PackageVersion: 1.1.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: bun-windows-x64\bun.exe
+  PortableCommandAlias: bun

```

I forgot about this too, a `bunx` alias for `bun x` (Bun automatically detects the naming):
```suggestion
  PortableCommandAlias: bun
- RelativeFilePath: bun-windows-x64\bun.exe
  PortableCommandAlias: bunx
```

---

- https://github.com/microsoft/winget-pkgs/pull/147295